src/s3fs_help.cpp

```diff
@@ -31,7 +31,7 @@
 //-------------------------------------------------------------------
 // Contents
 //-------------------------------------------------------------------
-static const char help_string[] = 
+static const char help_string[] =
```

Comment:
Please remove _all_ noisy formatting changes from all files.  This ensures that the git log remains easy to understand.  You can submit these kinds of cleanup in a separate PR, preferably machine-generated by `sed` or `clang-tidy`.